### ISSUE TITLE
Compilation error

### ISSUE THREAD

**Issue by gkdevone** [stutrek/scrollmonitor-hooks#1]
when we just import withScrollContainer

`import {withScrollContainer} from "scrollmonitor-hooks";`

we get compilation error below:

./node_modules/scrollmonitor-hooks/index.js
SyntaxError: /Users/geokal/Projects/ggking-webui/node_modules/scrollmonitor-hooks/index.js: Unexpected token (98:8)

   96 | 	const ref = useRef(null);
   97 | 	const container = useScrollContainer(ref);
>  98 | 	return <ScrollContainerContext.Provider value={container}>
      | 	       ^
   99 | 		<Component ref={ref} {...props} />
  100 | 	</ScrollContainerContext.Provider>;
  101 | };

environment:
    "react": "^16.8.6",
    "react-dom": "^16.8.6",
    "react-scripts": "^2.1.8",

**Comment by sallar** [comment]
There's a mistake in this package's config. But for now you can fix it like so:

```
import { useScrollState } from "scrollmonitor-hooks/dist/index.js"
```

**Comment by stutrek** [comment]
Sorry about this, it's fixed in 1.0.2.
